### PR TITLE
[git] Expose git.pushForce in the UI

### LINKED ISSUE #62020: [git] Expose git.pushForce in the UI
Since we already have the command in the pallet, it would be nice to expose it in the UI. 

Here:
screenshot 2018-10-29 at 09 51 10

### PR DESCRIPTION
**Pull request by oneslash**
fixes: #62020

### CODE REVIEW

File: extensions/git/package.json [hunk 1]
```diff
@@ -657,6 +657,11 @@
           "group": "1_sync",
           "when": "scmProvider == git"
         },
+        {
+          "command": "git.pushToForce",
+          "group": "1_sync",
+          "when": "scmProvider == git"
```
Comment (joaomoreno) [inline extensions/git/package.json]: You'll want to add `&& config.git.allowForcePush` to this `when` condition.

**Review by joaomoreno — CHANGES_REQUESTED** [review]

**Comment by oneslash** [comment]
@joaomoreno, thanks for the CR, the fix is done.

**Comment by oneslash** [comment]
@joaomoreno, tests are failing but I don't see how my change might cause it

**Comment by joaomoreno** [comment]
Thanks! :beers:

**Pull request merged**